model/src/datetime/mod.rs

```diff
@@ -266,28 +233,6 @@ impl Serialize for Timestamp {
     }
 }
 
-impl TryFrom<&'_ str> for Timestamp {
```

Comment:
its not an antipattern to implement both tryfrom and fromstr

Reply:
I'd say it is since std doesn't have any `TryFrom<&str>` https://doc.rust-lang.org/stable/std/convert/trait.TryFrom.html#implementors

Reply:
https://github.com/rust-lang/api-guidelines/discussions/182

https://rust-lang.github.io/api-guidelines/interoperability.html#c-conv-traits